scavenger-api/src/main/kotlin/com/navercorp/scavenger/controller/ExportController.kt

```diff
@@ -0,0 +1,48 @@
+package com.navercorp.scavenger.controller
+
+import com.navercorp.scavenger.service.ExportSnapshotMethodService
+import org.springframework.core.io.InputStreamResource
+import org.springframework.core.io.Resource
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import java.io.ByteArrayInputStream
+import java.io.ByteArrayOutputStream
+
+@RestController
+@RequestMapping("/api")
+class ExportController(
```

Comment:
I think it's better to move export feature to SnapshotController.
We organized the controllers by domain not featrue.

Reply:
Controllers by domain means according to entities and models?
I tried to fix this on commit 11a7c7e, am I understand correctly?